app/src/main/res/layout/fragment_app_intro.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/main"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:paddingTop="@dimen/appintro_statusbar_height"
+  android:paddingBottom="@dimen/appintro2_bottombar_height"
```

Comment:
Can we also add some horizontal padding? I noticed one heading fitting itself right to the edges.

Reply:
Sure, I think 20dp of horizontal padding is enough, right?
```
  android:paddingStart="20dp"
  android:paddingEnd="20dp"
```

Reply:
yeah, 20 is plenty.